gh-121528: Fix the process crash when load extension when it's compiled with low version of the python. (#121550)
python/cpython: Misc/NEWS.d/next/Core_and_Builtins/2024-07-10-03-13-19.gh-issue-121528._Bo7rw.rst

> Fix the process crash when load extension when it's compiled with low

Reviewer: I think this should be reworded, it's not too clear to someone who's not familiar with the exact issue.

```suggestion
Fix crash when loading extension modules that are compiled with an older
```

> version of the python.

Reviewer: ```suggestion
version of Python.
```
